src/dcurl.h

```diff
@@ -4,8 +4,44 @@
 #include <stdbool.h>
 #include "trinary.h"
 
+/**
+ * @file dcurl.h
+ */
```

Comment:
Add general descriptions about dcurl here.

Reply:
Done.

Reply:
Did you mean the general descriptions about dcurl rather than the description of this file?
I think I might update the comment with the latter one.

Reply:
It would be something describing the functionality about dcurl.
> dcurl library is provided as hardware-accelerated implementation for IOTA PearlDiver, which utilizes multi-threaded SIMD, FPGA and GPU.

Reply:
Done.